src/definition.cpp

```diff
@@ -120,7 +120,7 @@ void DefinitionImpl::Private::setDefFileName(const QCString &df)
 void DefinitionImpl::Private::init(const QCString &df, const QCString &n)
 {
   setDefFileName(df);
-  QCString lname = n;
+  const QCString &lname = n;
```

Comment:
Looks like there is no reason (anymore) to have `lname`, so best use `n` everywhere instead..